src/logic/addressBook/model/addressBook.ts

```diff
@@ -1,17 +1,28 @@
+import { ETHEREUM_NETWORK } from 'src/config/networks/network.d'
+import { getNetworkId } from 'src/config'
+
+export const ADDRESS_BOOK_DEFAULT_NAME = 'UNKNOWN'
+
 export type AddressBookEntry = {
-  address: string
-  name: string
+  address: string // the contact address
+  name: string // human-readable name
+  chainId: ETHEREUM_NETWORK // see https://chainid.network
 }
 
+const networkId = getNetworkId()
+
 export const makeAddressBookEntry = ({
   address = '',
```

Comment:
is this allowed for compatibility? Are we going to allow creating entries with empty address and name?

Reply:
not sure at all, this may serve as a skeleton for a particular case (?). I just left it as it was.

Reply:
It should definitely be a required arg in this fn.